UI/index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
```

Comment:
I wonder whether the index.html is still needed, since we have already the index.erb file in place.